src/NeuralNetwork/NeuralNetwork.js

```diff
@@ -251,7 +251,7 @@ class NeuralNetwork {
       const modelJsonResult = await axios.get(filesOrPath.model, {
         responseType: "text",
       });
-      const modelJson = JSON.stringify(modelJsonResult.data);
+      const modelJson = modelJsonResult.data;
       // TODO: browser File() API won't be available in node env
       const modelJsonFile = new File([modelJson], "model.json", {
```

Comment:
TBH this entire section of the code feels a bit odd and unnecessary.  We are fetching the files, creating `File` objects, and then passing them to the TF model with `tf.io.browserFiles`.  But TF is definitely capable of fetching files on its own so it seems like we shouldn't have to do that on our end.  It seems like we should just be able to pass the URLs.  That's a discussion for another time though, because maybe there's a reason that this is necessary and I'm just not seeing it.

Reply:
I believe this is necessary due to the way that the p5.js web editor stores the model files.

Reply:
Yeah that makes sense.  I know the p5.js web editor well and the way that it handles uploaded files is complicated and messy because there is a URL that the user sees and uses in their code, `"/model/model.json"`, and then there is the actual URL of the file which is some long string file name on an S3 server.  So when TF is looking for other files with specific names in the same directory it's not gonna find them.  We might be able to define a custom TF loader, but it's not as simple as I was thinking.